doc/tutorial/network_ring_gpu.rst

```diff
@@ -0,0 +1,126 @@
+some rules. Arbor provides :class:`arbor.partition_load_balance`, which, using the recipe and execution
+context, creates the :class:`arbor.domain_decomposition` object for us.
+
+A way to customize :class:`arbor.partition_load_balance` is by providing a :class:`arbor.partition_hint`. They let
+you configure how cells are distributed over the resources in the :class:`~arbor.context`, but without requiring you
+to know the precise configuration of a :class:`~arbor.context` up front. Whether you run your simulation on your 
+laptop CPU, desktop GPU, CPU cluster of GPU cluster, using :class:`partition hints<arbor.partition_hint>` you can
+just say: use GPUs, if available. You only have to change the :class:`~arbor.context` to actually define which 
+hardware Arbor will execute on.
+
+Step **(14)** creates a :class:`arbor.partition_hint`, and tells it to put 1000 cells in a groups allocated to GPUs, 
+and to prefer the utilization of the GPU if present. Lastly, a dictionary is created with which hints are assigned 
+to a particular :class:`arbor.cell_kind`. Different kinds may favor different execution, hence the option. In this
+simulation, there are only :class:`arbor.cell_kind.cable`, so we assign the hint to that kind.
+
+Step **(15)** creates a :class:`arbor.partition_load_balance` with the recipe, context and hints created above.
+Another checkpoint will help us understand how long creating the load balancer took.
+
+.. literalinclude:: ../../python/example/network_ring_gpu.py
+   :language: python
+   :lines: 138-148
+
+
+The simulation
+**************
+
+Step **(16)** creates a :class:`arbor.simulation`, sets the spike recorders to record, creates a :term:`handle` to their eventual results and makes another checkpoint.
+
+.. literalinclude:: ../../python/example/network_ring_gpu.py
+   :language: python
+   :lines: 150-154
+
+The execution
+*************
+
+Step **(17)** runs the simulation. Since we have more cells this time, which are connected in series, it will take some time for the action potential to propagate. In the :ref:`ring network <tutorialnetworkring>` we could see it takes about 5 ms for the signal to propagate through one cell, so let's set the runtime to ``5*ncells``. Then, another checkpoint, so that we'll know how long the simulation took.
+
+.. literalinclude:: ../../python/example/network_ring_gpu.py
+   :language: python
+   :lines: 156-159
+
+The results
+***********
+
+The scientific results should be similar, other than number of cells, to those in :ref:`ring network <tutorialnetworkring>`,
+so we'll not discuss them here. Let's turn our attention to the :class:`~arbor.meter_manager`.
+
+.. literalinclude:: ../../python/example/network_ring_gpu.py
+   :language: python
+   :lines: 161-163
+
+Step **(18)** shows how :class:`arbor.meter_report` can be used to read out the :class:`~arbor.meter_manager`. It generates a table with the time between checkpoints. For example:
```

Comment:
The output below looks different for GPUs, no?

Reply:
Right, the example output here is for one config, in this case my laptop CPU.

Reply:
Since we advertise GPU stuff in the example that is worth commenting.